app/main.dev.ts

```diff
@@ -123,6 +124,11 @@ app.on( 'ready', async () => {
             if ( commandLine.length >= 2 && uri ) {
                 onReceiveUrl( store, uri );
             }
+
+            if ( commandLine.includes( '--trigger-update' ) ) {
```

Comment:
maybe even we need a global var for this, as this state could be different to process.argv etc, no?